package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fullstory/data-layer-observer",
-  "version": "2.0.1",
+  "version": "2.1.1",
```

Comment:
hmm... now that I'm seeing this here, did an `npm version` command bump from `2.0.1` to `2.1.1`?

Reply:
I did this. 🤦‍♂️ I'll get this updated to 2.1.0 before merging.